scripts/install

```diff
@@ -34,20 +46,14 @@ ynh_add_nginx_config
 # BUILD APP
 #=================================================
 
-cd $install_dir
-ynh_exec_warn_less git clone https://github.com/terraforming-mars/terraforming-mars.git
-
-cd $install_dir/terraforming-mars
-ynh_exec_warn_less git checkout $TM_version
-
-pushd $install_dir/terraforming-mars
+pushd $install_dir
     ynh_use_nodejs
 
 	ynh_script_progression --message="npm install... This can be very long, be patient !" --weight=18
-	ynh_exec_warn_less env $ynh_node_load_PATH $ynh_npm install --network-timeout=100000
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH $ynh_npm ci --quiet --production --prefer-offline --network-timeout=100000
 
 	ynh_script_progression --message="npm run build... This can be very long, be patient !" --weight=25
-	ynh_exec_warn_less env $ynh_node_load_PATH $ynh_npm run build
+	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH $ynh_npm run build
```

Comment:
```suggestion
	export SOURCE_VERSION="$(cat YNH_APP_BASEDIR/manifest.toml | toml_to_json | jq -r ".resources.sources.main.url" | awk -F/ '{ print $NF }')"
	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH $ynh_npm run build
```

Looking at https://github.com/terraforming-mars/terraforming-mars/blob/main/make_static_json.js#L54, the build will assume it's running inside a git repo unless we define SOURCE_VERSION as the current commit, which we can obtain by parsing the manifest.toml